### DISCUSSION TITLE
update语法问题

### DISCUSSION THREAD

**Discussion by supergenge** [apache/doris#18381] [RFC]
版本号：doris-1.2.2-rc01
执行关联更新：
update test_wyg02 t2 set t2.value=t1.aa*10
FROM test_wyg01 t1 where t1.id=t2.id
错误提示：
org.jkiss.dbeaver.model.sql.DBSQLException: SQL 错误 [1105] [HY000]: errCode = 2, detailMessage = Syntax error in line 1:
...sql> */ update test_wyg02 t2 set t2.value=t1.aa*10

                             ^
Encountered: IDENTIFIER
Expected

按照官方文档写的update语句，提示语法错误，请问是什么原因？

**Comment by zy-kkk** [comment]
This is not supported in version 1.2 and will be supported in a future 2.0 release
![image](https://user-images.githubusercontent.com/70795751/230282707-9834bc41-a85c-4525-9af2-5bd7572bc25b.png)
